src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -90,10 +90,10 @@ public void delete(@PathVariable Long id) {
 	 */
 	@PutMapping("/{id}")
 	public Book updateBook(@RequestBody Book book, @PathVariable Long id) {
-		if (book.getId() != id) {
-			throw new BookIdMismatchException(NotificationCode.MISMATCH);
+		if (book.getId() == id) {
```

Comment:
If you change != to ==, this will raise an exception when the id of the book you want to modify is the same of the id that you receive on the RequestBody. That does not seem right, because you do want to modify the Book if and only if the id of the path matches the id of the RequestBody.

Reply:
finger error :)